packages/browser/src/integrations/browserapierrors.ts

```diff
@@ -169,30 +163,25 @@ function _wrapXHR(originalSend: () => void): () => void {
 }
 
 function _wrapEventTarget(target: string): void {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const globalObject = WINDOW as { [key: string]: any };
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-  const proto = globalObject[target] && globalObject[target].prototype;
+  const globalObject = WINDOW as unknown as Record<string, { prototype?: object }>;
+  const targetObj = globalObject[target];
+  const proto = targetObj && targetObj.prototype;
 
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, no-prototype-builtins
-  if (!proto || !proto.hasOwnProperty || !proto.hasOwnProperty('addEventListener')) {
+  if (!proto || Object.prototype.hasOwnProperty.call(proto, 'addEventListener')) {
```

Comment:
is there a reason not to use `Object.prototype.hasOwnProperty` here, but use the prototype method itself? 🤔

Reply:
what's the difference between the two? shouldn't this be equal since we pass `proto` as the `this` arg? I guess the only apparent reason to me would be saving a couple of bytes but if using `Object.prototype.hasOwnProperty` is more correct, we should go for it!

Reply:
Well there is https://eslint.org/docs/latest/rules/no-prototype-builtins which is "fixed" by this, not sure how important this is though 🤔

Reply:
Hmm, so there does seem to be a difference there 🤔 changing this changes the browser integration tests, `target` of the mechanism becomes `Window` or `Node` instead of `EventTarget` 🤔 not sure why, but I guess I'll leave this as it was then!

Reply:
TIL, thanks! No objections to leaving it as-is :D